content/S2016/S16-event-teaparty.md

```diff
@@ -0,0 +1,24 @@
+title: Tea Party and Volunteer Meeting
+Date: 2016-4-27
```

Comment:
Nit: 04? I don't know if `-4-` will parse properly, although if you spin up a review app you can test this.